Homework_No1.txt

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE new_agents
+SELECT * FROM`agents` 
```

Comment:
Insert records into 'new_agents' table from 'agents' table.
-> we should use INSERT INTO SELECT in here.

Reply:
Made a change. Thank you for confirmation.